ViewModel/Hydraulics/UnsteadyHDF/UnsteadyHDFImporterVM.cs

```diff
@@ -186,6 +186,12 @@ public void FileSelected(string fullpath)
             }
         }
 
+        private string getUnsteadyRASResultName(string file)
+        {
+            RasMapperLib.RASResults result = new RasMapperLib.RASResults(file);
+            return result.PlanAttributes.PlanTitle;
```

Comment:
not sure what RasResults does here. If it can return null then we should probably add a null check here.